ports/unix/variants/mpconfigvariant_common.h

```diff
@@ -91,7 +91,11 @@
 #define MICROPY_PY_OS_ERRNO            (1)
 #define MICROPY_PY_OS_GETENV_PUTENV_UNSETENV (1)
 #define MICROPY_PY_OS_SEP              (1)
+#if defined(__wasi__)
```

Comment:
Similar comment as above: instead of modifying this variant it would be better to create it separately